source/_meta/_00-head.twig

```diff
@@ -23,5 +22,5 @@
     <!-- End Pattern Lab -->
 
   </head>
-  <body class="{{ body_class }}">
+  <body class="{{ body_class }}" id="top">
```

Comment:
We also need to add this id to the `_02-basic-head.twig` template.

Reply:
Ah, didn't know about that one. Will update.